opal/mca/pmix/pmix2x/pmix/src/mca/gds/ds12/gds_dstore.c

```diff
@@ -2804,6 +2808,16 @@ static pmix_status_t dstore_assign_module(pmix_info_t *info, size_t ninfo,
         }
     }
 
+<<<<<<< c632784ca34c467055eadcb4efe84a25e5a3911b:opal/mca/pmix/pmix2x/pmix/src/dstore/pmix_esh.c
```

Comment:
not sure that last part is true, but no matter - it runs just fine and passed an MTT scan, so I think things resolved correctly. Anyway, I'm updating from tarball now.